snuba_sdk/query_visitors.py

```diff
@@ -171,14 +199,32 @@ def _visit_totals(self, totals: Optional[Totals]) -> str:
             return f"TOTALS {self.translator.visit(totals)}"
         return ""
 
+    def _visit_consistent(self, consistent: Optional[Consistent]) -> str:
+        return str(consistent.value) if consistent is not None else ""
```

Comment:
This is what I meant before. We should either provide a consistent flag if it is true or no consistent flag at all. Adding a third state adds complexity that does not seems needed.